src/main/java/cs/f10/t1/nursetraverse/model/ModelState.java

```diff
@@ -0,0 +1,14 @@
+package cs.f10.t1.nursetraverse.model;
+
+/**
+ * Enum that indicates the current state of the model. This state is used to determine whether certain commands
+ * are available for use.
+ */
+public enum ModelState {
```

Comment:
Would it be clearer to name this class `ModelStatus` or something similar? At first glance it may seem like 'state' refers to the actual data of the model.